{{cookiecutter.project_slug}}/.github/workflows/main.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - main
     paths-ignore:
-      - CHANGES.rst
```

Comment:
Don't ignore CHANGES.rst?

Reply:
There is no `CHANGES.rst` in the generated package, we renamed this to `HISTORY.rst` at some point.